contracts/v0.8/tests/bigints.test.sol

```diff
@@ -30,7 +30,7 @@ import "../utils/BigInts.sol";
 contract BigIntsTest {
     using BigInts for CommonTypes.BigInt;
 
-    function to_uint256() public view {
+    function test_to_uint256() public view {
```

Comment:
Can we rename this back to `to_uint256`.
The reason is this is not an actual test, but a test contract that will be called from rust tests.
You can find the actual test that does the assertions here:
https://github.com/filecoin-project/filecoin-solidity/blob/master/testing/tests/bigints.rs

The way you have added the test case will make it run by default because the rust test will call the method `to_int256_negative`.

Reply:
done

Reply:
BTW, how do you run `forge test` without test_ prefix in test funcs?

Reply:
These tests are not run with `forge`, but with `cargo`, since they are rust tests, the contracts are called from rust.
Check the Makefile, `test_integration` target for example. You can update the make target to run your specific test file while testing, and later on validate on the whole test suite.

Reply:
Actually, just run `make test_bigints`. There is already a target for that.